nilearn/datasets/utils.py

```diff
@@ -527,21 +530,8 @@ def _fetch_file(url, data_dir, resume=True, overwrite=False,
         shutil.move(temp_full_name, full_name)
         dt = time.time() - t0
         if verbose > 0:
-            print('...done. (%i seconds, %i min)' % (dt, dt // 60))
-    except _urllib.error.HTTPError as e:
-        if verbose > 0:
-            print('Error while fetching file %s. Dataset fetching aborted.' %
-                  (file_name))
-        if verbose > 1:
-            print("HTTP Error: %s, %s" % (e, url))
-        raise
-    except _urllib.error.URLError as e:
-        if verbose > 0:
-            print('Error while fetching file %s. Dataset fetching aborted.' %
-                  (file_name))
-        if verbose > 1:
-            print("URL Error: %s, %s" % (e, url))
-        raise
```

Comment:
These errors are re-raised, so outputting here at such a low verbosity level causes code that expects and handles errors to have error output. I suggest it should either be at a higher verbosity level or be removed, and I think removed is better.